tests/test_sanity_check.py

```diff
@@ -216,6 +228,7 @@ def test_global_free_space_percentage(self):
         )
         self.assertEqual(report, responses)
 
+
```

Comment:
@UmanShahzad Might be unnecessary change (unless it was pep8 complaining about something)

Reply:
Unfortunately we don't have pep8 (or any quality check) on this repository's Python code, but this is definitely something it'd complain about. I don't want to expand this PR so much as to include pep8 whole, but I noticed this so added it -- leaving the code just a bit better than I found it.